From b84bd545988c30296209b80a60f33782254a8f56 Mon Sep 17 00:00:00 2001
From: Arian Sanusi <arian@sanusi.de>
Date: Thu, 30 Jul 2009 18:40:03 +0200
Subject: [PATCH 11/19] add suspend quirks for HP tc4400, nc4400, tc4200

Added suspend quirks for HP tc4400, nc4400, tc4200. They all
need vbe-post.
---
 .../10freedesktop/20-video-quirk-pm-hp.fdi         |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi b/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi
index cd8f41e..4a3016c 100644
--- a/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi
+++ b/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi
@@ -121,7 +121,7 @@
         <merge key="power_management.quirk.s3_bios" type="bool">true</merge>
         <merge key="power_management.quirk.vbemode_restore" type="bool">true</merge>
       </match>
-      <match key="system.hardware.product" contains_outof="nx6120;nc4200;HP 530 Notebook PC">
+      <match key="system.hardware.product" contains_outof="tc4400;nc4400;tc4200;nx6120;nc4200;HP 530 Notebook PC">
         <merge key="power_management.quirk.vbe_post" type="bool">true</merge>
       </match>
 
-- 
1.6.4.4

